From 33145774d9d41ac306f972e0247c9a073d5dbfc9 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Fri, 14 Jul 2017 18:28:28 +0200
Subject: [PATCH] build-sys: install udev rule 70-joystick.{rules,hwdb} (#6363)

* meson: install udev files 70-joystick.{rules,hwdb}
* Makefile: install udev file 70-joystick.hwdb

(cherry picked from commit 816be2ba448940e2517dba81492e80b1e6a5954f)
---
 Makefile.am       | 1 +
 hwdb/meson.build  | 1 +
 rules/meson.build | 1 +
 3 files changed, 3 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index c16e62280b..b95c93bb98 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4062,6 +4062,7 @@ dist_udevhwdb_DATA = \
 	hwdb/60-evdev.hwdb \
 	hwdb/60-keyboard.hwdb \
 	hwdb/60-sensor.hwdb \
+	hwdb/70-joystick.hwdb \
 	hwdb/70-mouse.hwdb \
 	hwdb/70-pointingstick.hwdb \
 	hwdb/70-touchpad.hwdb
diff --git a/hwdb/meson.build b/hwdb/meson.build
index 74a93f9ccb..6fceff2b3b 100644
--- a/hwdb/meson.build
+++ b/hwdb/meson.build
@@ -12,6 +12,7 @@ hwdb_files = files('''
         60-evdev.hwdb
         60-keyboard.hwdb
         60-sensor.hwdb
+        70-joystick.hwdb
         70-mouse.hwdb
         70-pointingstick.hwdb
         70-touchpad.hwdb
diff --git a/rules/meson.build b/rules/meson.build
index 0f818a506f..7f4725ad65 100644
--- a/rules/meson.build
+++ b/rules/meson.build
@@ -12,6 +12,7 @@ rules = files('''
         60-sensor.rules
         60-serial.rules
         64-btrfs.rules
+        70-joystick.rules
         70-mouse.rules
         70-touchpad.rules
         75-net-description.rules
